deploy/src/native_to_erc/home.js

```diff
@@ -57,10 +63,19 @@ if (isRewardableBridge) {
   VALIDATORS_REWARD_ACCOUNTS = env.VALIDATORS_REWARD_ACCOUNTS.split(' ')
 }
 
-async function initializeBridge({ validatorsBridge, bridge, initialNonce }) {
+async function initializeBridge({ validatorsBridge, bridge, limitsContract, initialNonce }) {
   let nonce = initialNonce
   let initializeHomeBridgeData
 
+  const executionLimitsArray = RELATIVE_DAILY_LIMIT
+    ? [TARGET_LIMIT, THRESHOLD, FOREIGN_MAX_AMOUNT_PER_TX, FOREIGN_MIN_AMOUNT_PER_TX]
+    : [FOREIGN_DAILY_LIMIT, FOREIGN_MAX_AMOUNT_PER_TX, FOREIGN_MIN_AMOUNT_PER_TX]
+
+  const RELATIVE_DAILY_LIMIT_PARAMS = `TARGET_LIMIT: ${TARGET_LIMIT} which is ${
+    Web3Utils.fromWei(Web3Utils.toBN(TARGET_LIMIT).mul(Web3Utils.toBN(100)))
+  }%,
+  THRESHOLD: ${THRESHOLD} which is ${Web3Utils.fromWei(THRESHOLD)} in eth,`
```

Comment:
If `RELATIVE_DAILY_LIMIT=false` the variables `TARGET_LIMIT` and `THRESHOLD` could not be defined in `.env` file, in that case, `Web3Utils.fromWei(Web3Utils.toBN(TARGET_LIMIT)` and `Web3Utils.fromWei(THRESHOLD)` fails with an error. The same applies to the scripts of the other modes. I think a possible solution could be to only define `RELATIVE_DAILY_LIMIT_PARAMS` if `RELATIVE_DAILY_LIMIT=true`